vars/checkPatch.groovy

```diff
@@ -41,11 +60,23 @@ def call(Map config) {
                  context: "pre-build" + "/" + env.STAGE_NAME,
                  status: "PENDING"
 
-    script = 'GH_USER="' + config['user'] + '"' + \
-             ' GH_PASS="' + config['password'] + '"' + \
-             ' CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
+    rc = 1
+    script = 'CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
              ' jenkins/code_review/jenkins_github_checkwarn.sh'
-    rc = sh(script: script, returnStatus: true)
+
+    if (config['review_creds']) {
+      withCredentials([[$class: 'UsernamePasswordMultiBinding', 
+                      credentialsId: config['review_creds'],
+                      usernameVariable: 'GH_USER',
+                      passwordVariable: 'GH_PASS']]) {
+        rc = sh(script: script, returnStatus: true)
+      }
+    } else {
+      // Older method using username/password
+      script = 'GH_USER="' + config['user'] + '"' + \
+        ' GH_PASS="' + config['password'] + '"' + script
```

Comment:
I think you need a space before `+ script`
```suggestion
        ' GH_PASS="' + config['password'] + '" ' + script
```

Reply:
Fixed in next push.